corehq/apps/app_manager/dbaccessors.py

```diff
@@ -22,14 +22,14 @@ def domain_has_apps(domain):
     return len(results) > 0
 
 
-def get_latest_released_app_doc(domain, app_id, min_version=None):
+def get_latest_released_app_doc(domain, app_id):
```

Comment:
I'm not totally sure about this, so let me talk through it out loud and then you can let me know if you're seeing something I'm not.

In this function (`get_latest_released_app_doc`) `app_id` means the "working copy" within whose build to search; i.e. `app_id` cannot be a build id. I reach this conclusion, because if `app_id` were a build id, it would be nonsensical in the view key, because the operative line in the `map.js` is
```js
emit(['^ReleasedApplications', doc.domain,  doc.copy_of, doc.version], null);
```
and nothing can have a `copy_of` that equals a build id.

And `min_version` is a version number (e.g. 59, 296, 315).

Thus when it's called below (the only time it's called with `min_version` set), it would appear that `min_version` is in fact doing something important. `parent_app_id` would again, I believe, be an app "working copy" and not a build id.

Let me know if that makes sense! And let me know if I'm missing something.